### DISCUSSION TITLE
How to delete/manage envs after the project folder is deleted?

### DISCUSSION THREAD

**Discussion by Justin1904** [python-poetry/poetry#9165] [Q&A]
Sometimes I work on a project that I set up with poetry. Later I deleted that project locally from my computer. How do I locate/delete/manage the poetry-created envs associated with the deleted project? [Docs on managing envs](https://python-poetry.org/docs/managing-environments/) talks about `poetry env` which only works when the project folder still exists.

**Comment by radoering** [comment]
You can only get the folder where the envs for all your projects are stored: `poetry config virtualenvs.path`
